src/Common/xrpl-network.ts

```diff
@@ -1,14 +1,3 @@
-/**
- * XRP Ledger Networks.
- *
- * @deprecated Use XrplNetwork.
- */
-export enum XRPLNetwork {
-  Dev = 'devnet',
-  Test = 'testnet',
-  Main = 'mainnet',
-}
-
 /**
```

Comment:
You can actually just delete this entire file. I moved `XrplNetwork` into xpring-common-js so you can just pull it from there now that we don't have a shim layer.